pkg/kubelet/stats/helper.go

```diff
@@ -61,6 +61,7 @@ func cadvisorInfoToCPUandMemoryStats(info *cadvisorapiv2.ContainerInfo) (*statsa
 			UsageBytes:      &cstat.Memory.Usage,
 			WorkingSetBytes: &cstat.Memory.WorkingSet,
 			RSSBytes:        &cstat.Memory.RSS,
+			SwapBytes:       &cstat.Memory.Swap,
```

Comment:
We are already getting this data from cadvisor so we don't need to do anything special, it should come with the rest of our memory stats already: https://github.com/google/cadvisor/blob/b8e5d34f2357154259c71288663bf79b2cff7c31/container/libcontainer/handler.go#L801-L845

Reply:
delete this field directly? @ehashman